als.py

```diff
@@ -58,14 +58,26 @@ def __init__(self, base_path, server_address, RequestHandlerClass=HTTPHandler):
         self.base_path = base_path
         BaseHTTPServer.__init__(self, server_address, RequestHandlerClass)
 
-class StoppableThread(threading.Thread):
+class StoppableServerThread(threading.Thread):
     """Thread class with a stop() method. The thread itself has to check
     regularly for the stopped() condition."""
 
-    def __init__(self):
-        super(StoppableThread, self).__init__()
+    def __init__(self, web_dir):
+        # web stuff
+        self.web_dir = web_dir
+        self.httpd = HTTPServer(self.web_dir, ("", 8000))
```

Comment:
Would you mind making the port number user configurable, with a default value of 8000 ?

To do that, you'll have to do a rebase of your branch on alpha

Reply:
Good point @deufrai ! 
I should also take care of the ui counterpart (for the port) + config save. I'll try to tackle this within this week, thank you for the suggestion

Reply:
config is saved just before app quits
you'll just have to add the port number to our config (with a new setter function in config.py)